dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceApplication.java

```diff
@@ -702,4 +715,41 @@ private void restrictSourceFiles(DockstoreWebserviceConfiguration configuration)
             SourceFile.unrestrictPaths();
         }
     }
+
+    private static class LogPrivilegedEndpointsListener implements ApplicationEventListener {
+
+        @Override
+        public void onEvent(ApplicationEvent event) {
+            if (event.getType() == ApplicationEvent.Type.INITIALIZATION_APP_FINISHED) {
+                List<String> roles = List.of("admin", "curator", "platformPartner");
```

Comment:
Maybe use the `ADMIN` constant in https://github.com/dockstore/dockstore/blob/1.16.0-alpha.10/dockstore-webservice/src/main/java/io/dockstore/webservice/SimpleAuthorizer.java to remind future editors of one place to (consider) editing the others.